src/components/Tables/DomainTable/DomainTable.tsx

```diff
@@ -81,21 +81,17 @@ const DomainTable: React.FC<DomainTableProps> = ({
 }) => {
 	const { mvpVersion } = useMvpVersion();
 	const { getBidsForDomain } = useBidProvider();
-
-	const [hasMetadataLoaded, setHasMetadataLoaded] = useState(false);
-	const [hasBidDataLoaded, setHasBidDataLoaded] = useState(false);
 	const [isLoading, setIsLoading] = useState(true);
-
 	const [containerHeight, setContainerHeight] = useState(0);
 	const [searchQuery, setSearchQuery] = useState('');
-
 	const [modal, setModal] = useState<Modals | undefined>();
 	const [biddingOn, setBiddingOn] = useState<DisplayDomain | undefined>();
-
-	const [loadedDomains, setLoadedDomains] = useState<DomainData[]>([]);
+	const [data, setData] = useState<DomainData[]>([]);
+	const [tableData, setTableData] = useState<DomainData[]>([]);
+	const [isSubscribed, setIsSubscribed] = useState(false);
 
 	const containerRef = useRef<HTMLDivElement>(null);
-
+	const chunkLength = 5;
```

Comment:
Why does chunk length matter, leave a comment here to describe the purpose of this variable